src/useTubular.ts

```diff
@@ -71,7 +71,7 @@ export const useTubular = (
             }
 
             const payload = allRows
-                ? (await getAllRecords()(new GridRequest(tbState.columns, -1, 0, tbState.searchText))).payload
+                ? (await getAllRecords()(new GridRequest(tbState.columns, 1000000, 0, tbState.searchText))).payload
```

Comment:
This is not needed. We have addressed this issue at https://github.com/unosquare/tubular-common/pull/347/files#diff-d0a0ca5370b12b88e923d3279fd11385944530ff630be9fa2637d762d0ef24fbR79.